src/pages/Home.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Home() {
```

Comment:
same comment as the `ForgotPassword` component, and it should apply to SignIn and Signup component